be/src/exec/vectorized/aggregate/aggregate_streaming_node.cpp

```diff
@@ -240,14 +240,20 @@ std::vector<std::shared_ptr<pipeline::OperatorFactory> > AggregateStreamingNode:
     // shared by sink operator factory and source operator factory
     AggregatorFactoryPtr aggregator_factory = std::make_shared<AggregatorFactory>(_tnode);
 
+    // Create a shared RefCountedRuntimeFilterCollector
+    auto&& rc_rf_probe_collector = std::make_shared<RcRfProbeCollector>(2, std::move(this->runtime_filter_collector()));
     auto sink_operator = std::make_shared<AggregateStreamingSinkOperatorFactory>(context->next_operator_id(), id(),
                                                                                  aggregator_factory);
+    // Initialize OperatorFactory's fields involving runtime filters.
+    this->init_runtime_filter_for_operator(sink_operator.get(), context, rc_rf_probe_collector);
```

Comment:
Needn't handle runtime filter in aggregate or distinct streaming_node

Reply:
its a common interface for framework, if a operator is not runtime filter consumer, then it has no runtime fitler evaluation interpolation.